.buildkite/scripts/pipelines/chromium_linux_build/issue_feedback/entry.js

```diff
@@ -0,0 +1,327 @@
+        ],
+        { printToScreen: true }
+      );
+
+      const linuxBuildArtifact = await fg(`chromium-*_${arch}.*`);
+
+      assert(linuxBuildArtifact.length, 'No linux build artifacts found');
+
+      const match = linuxBuildArtifact.find((artifact) =>
+        RegExp(String.raw`${arch}\.zip`).test(artifact)
+      );
+
+      assert.ok(match, `No linux build artifacts found for ${arch}`);
+
+      const archiveChecksum = await getSha256Hash(match);
+
+      assert.strictEqual(
+        archiveChecksum,
+        readFileSync(`${parse(match).name}.sha256`, 'utf-8').split(/\s/)[0],
+        'Checksum mismatch'
+      );
+
+      const zip = new AdmZip(match);
+      zip.extractAllTo('.', true);
+
+      const binaryChecksum = await getSha256Hash(`headless_shell-linux_${arch}/headless_shell`);
+
+      config[`linux_${arch}`] = {
+        archiveFilename: match,
+        archiveChecksum,
+        binaryChecksum,
+      };
+    })
+  );
+
+  console.log('--Modifying paths.ts file\n');
+
+  await $(
+    'npx',
+    [
+      '--',
+      'jscodeshift',
+      '--extensions=ts',
+      '--parser=tsx',
+      '--fail-on-error',
+      `--transform`,
+      resolve(
+        buildRoot,
+        '.buildkite/scripts/pipelines/chromium_linux_build/issue_feedback/transform_path_file.js'
+      ),
+      '../paths.ts',
+      `--updateConfig=${JSON.stringify(config)}`,
+      `--chromiumRevision=${chromiumRevision}`,
+      `--chromiumVersion=${chromiumVersion}`,
+    ],
+    { printToScreen: true }
+  );
+
+  await $('git', ['add', '../paths.ts']);
+
+  await $('git', ['commit', '-m', 'chore: update chromium paths']);
+
+  console.log('---Creating pull request \n');
+
+  // create clean branch based off of main
+
+  await $('git', ['checkout', 'main']);
+
+  await $('git', ['checkout', '-b', branchName]);
+
+  await $('git', ['cherry-pick', `${branchName}_temp~1`, `${branchName}_temp~0`], {
+    printToScreen: true,
+  });
+
+  await $('git', ['push', 'origin', branchName], {
+    printToScreen: true,
+  });
+
+  await $(
+    'gh',
+    [
+      'pr',
+      'create',
+      '--title',
+      prTitle,
+      '--body',
+      `Closes #${process.env.GITHUB_PR_NUMBER} \n\n This PR updates puppeteer to version ${process.env.PUPPETEER_VERSION} and updates the chromium paths to the latest known good version for windows and mac where the chromium revision is ${chromiumRevision} and version is ${chromiumVersion}, for linux a custom build was triggered to build chromium binaries for both x64 and arm64. \n **NB** This PR should be tested before merging it in as puppeteer might have breaking changes we are not aware of`,
+      '--base',
+      'main',
+      '--head',
+      branchName,
+      '--label',
+      'release_note:skip',
+      '--label',
+      'Team:SharedUX',
+    ],
+    {
+      printToScreen: true,
+    }
+  );
+
+  console.log('---Providing feedback to issue \n');
+
+  await $('ts-node', [
```

Comment:
give this a final run to verify if this is going to work. Sometimes these tool binary references won't be resolved correctly, and you'll need to relay through `yarn` or `npx`